update_prevalence.py

```diff
@@ -1103,7 +1109,7 @@ def parse_csv(cache: DataCache, model: Type[Model], url: str) -> List[Model]:
 
 def parse_json_list(cache: DataCache, model: Type[Model], url: str) -> List[Model]:
     print(f"Fetching {url}...", end=" ", flush=True)
-    result = pydantic.parse_obj_as(List[model], json.loads(cache.get(url)))
+    result = pydantic.parse_obj_as(List[model], json.loads(cache.get(url))) # type: ignore
```

Comment:
mypy wants `List[Model]` here, referencing the type instead of the variable as the type parameter; I don't know enough about pydantic to feel confident making that change.